src/logic/claims/data.test.ts

```diff
@@ -0,0 +1,133 @@
+import { calculateDeadline } from './data';
+import { addDays, addSeconds } from 'date-fns';
+
+describe('calculateDeadline', () => {
+  it('returns a deadline of 3 days for "ClaimCreated" status', () => {
+    const statusUpdatedAt = addSeconds(new Date(), -42);
+    const deadline = calculateDeadline({
+      status: 'ClaimCreated',
+      statusUpdatedAt,
+      dispute: null,
+    });
+
+    expect(deadline).toEqual(addDays(statusUpdatedAt, 3));
+  });
+
+  it('returns a deadline of 3 days for "SettlementProposed" status', () => {
+    const statusUpdatedAt = addSeconds(new Date(), -42);
+    const deadline = calculateDeadline({
+      status: 'SettlementProposed',
+      statusUpdatedAt,
+      dispute: null,
+    });
+
+    expect(deadline).toEqual(addDays(statusUpdatedAt, 3));
+  });
+
+  describe('when the dispute is in its "Evidence" period', () => {
+    it('returns the vote period end date', () => {
+      const periodChangedAt = addSeconds(new Date(), -42);
+      const evidencePeriod = 3.25 * 24 * 60 * 60;
```

Comment:
Just curious, why the decimal point numbers?

Reply:
That is the value on mainnet 👍